restcomm.android.sdk/src/main/java/org/restcomm/android/sdk/util/RCUtils.java

```diff
@@ -178,21 +181,21 @@ public static void validateConnectionParms(HashMap<String, Object> parameters) t
 
    }
 
-   public static boolean shouldUpdatePush(HashMap<String, Object> parameters) throws RCException{
-      if (!parameters.containsKey(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_FCM_SERVER_KEY) ||
-              parameters.get(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_FCM_SERVER_KEY) == null ||
-              TextUtils.isEmpty((String) parameters.get(RCDevice.ParameterKeys.PUSH_NOTIFICATIONS_FCM_SERVER_KEY))) {
-         throw new RCException(RCClient.ErrorCodes.ERROR_DEVICE_PUSH_NOTIFICATION_FCM_SERVER_KEY_MISSING);
-      }
-      return true;
-   }
-
-   public static boolean shouldRegisterForPush(HashMap<String, Object> parameters, StorageManagerInterface storageManagerInterface) throws RCException{
+   public static boolean validatePushSettings(HashMap<String, Object> parameters) throws RCException{
```

Comment:
Do we really need to return boolean here? I don't see us using the return value anywhere. If you want to be safe and make sure 'parameters' isn't null, you can add this check in the top level validation method and potentially introduce another error code for that, since parameters need to be provided both in RCDevice.initialize() and updateParams(). Wdyt?

Reply:
This will be removed in #749